cmd/sentinel/sentinel.go

```diff
@@ -400,6 +400,7 @@ func (s *Sentinel) setDBSpecFromClusterSpec(cd *cluster.ClusterData) {
 			db.Spec.FollowConfig.StandbySettings = clusterSpec.StandbySettings
 		}
 		db.Spec.AdditionalWalSenders = *clusterSpec.AdditionalWalSenders
+		db.Spec.AdditionalReplicationSlotNames = clusterSpec.AdditionalReplicationSlotNames
```

Comment:
This isn't needed and misleading since it's adding this to all the clusterdata dbs.

Since the replication slots are created only on the master instance (primary postgres instance or master postgres standby instance in a Standby stolon cluster) the keeper could just read the clusterspec.